symfony/framework-bundle/5.1/config/services.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+use App\Kernel;
+use Symfony\Component\DependencyInjection\Loader\Configurator as di;
+
+// This file is the entry point to configure your own services.
+// Files in the packages/ subdirectory configure your dependencies.
+
+return function (di\ContainerConfigurator $container, Kernel $kernel): void {
+
+    // Parameters are configuration that don't need to change depending on the machine where the app is deployed.
+    // see https://symfony.com/doc/current/best_practices/configuration.html#application-related-configuration
+    $container->parameters()
+    //    ->set(...)
+    ;
+
+    $services = $container->services()
+        ->defaults()
+            ->autowire()
+            ->autoconfigure()
+    ;
+
+    // Makes classes in src/ available to be used as services
+    $services
+        ->load('App\\', '../src/*')
```

Comment:
Since we use plain PHP, should it be better to use `__DIR__` ?

Reply:
Both work, it's a matter of preference, that's the very same in the end.

Reply:
Yes, but it's more clear IMHO

Reply:
updated